management/organization.go

```diff
@@ -20,7 +20,7 @@ type Organization struct {
 
 	// Metadata associated with the organization, in the form of an object with
 	// string values (max 255 chars). Maximum of 10 metadata properties allowed.
-	Metadata map[string]interface{} `json:"metadata,omitempty"`
+	Metadata *map[string]interface{} `json:"metadata,omitempty"`
```

Comment:
Isn't this a breaking change?

Reply:
That's correct, this is a breaking change, however we're still on v0 (https://github.com/auth0/go-auth0/releases/tag/v0.9.1) and we need a tristate in order to set this as empty in the terraform provider: https://github.com/auth0/terraform-provider-auth0/pull/245/files#diff-327e2a11c1ad2cd3d51588fb883dc109df5af1a48479a7bbbbe839e21a8f795eR203-R209

Reply:
Yes, I forgot the numeration changed when the repo moved over. My bad.

Reply:
Yes, I forgot the numeration changed when the repo moved over. My bad.

Reply:
HI @sergiught, Can you help me with how can I access data for this metadata field after the update? I am bit lost in this case.

Reply:
Hey @samirpdl 👋🏻 , have you tried something like `*organization.Metadata["some_key"]`? Considering now this is a pointer to a map, if we want to access the values we need to dereference with *.

Reply:
@sergiught , Thanks for your quick reply. I needed it on the user.UserMetadata, I tried the same way but I keep getting this error: `invalid operation: cannot index user.UserMetadata (variable of type *map[string]interface{})` I have been stuck for a long and then I reached out.

Reply:
@samirpdl 👋🏻 you need to surround the dereference in brackets:) e.g.

```go
package main

import (
	"fmt"
)

type User struct {
	Metadata *map[string]interface{}
}

func main() {
	metadata := map[string]interface{}{
		"status": "warrior",
	}
	someUser := &User{
		Metadata: &metadata,
	}

	fmt.Println((*someUser.Metadata)["status"])
}
```

Link to go playground to run this: https://go.dev/play/p/MrfpyX5r9U5

Reply:
Aaah, Silly me. Thanks @sergiught